.azure/pipelines/ci.yml

```diff
@@ -455,6 +455,9 @@ stages:
         includeForks: true
       - name: Linux_musl_x64_Packages
         path: artifacts/packages/
+      variables:
+      - name: skipComponentGovernanceDetection
+        value: true
```

Comment:
I'm not positive this will work. Kick of any of our public pipelines explicitly using your branch (so it's not a pull-request build), wait for the build to complete then make sure the expanded YAML includes the expected variable setting where the governance condition expands. Go to your build in AzDO and click "download logs" in the triple-dot menu then look at azure-pipelines-expanded.yaml in the Zip you download. If you use the default aspnetcore-ci pipeline, just need to check if the component governance step is included in the expansion of this particular job.

To get this working, might need to expand the parameter list as done at https://github.com/dotnet/aspnetcore/blob/5ac45e8c6f717bbd72a1702f7cd03920902259a1/eng/common/templates/job/job.yml#L16 then expand them as done at https://github.com/dotnet/aspnetcore/blob/5ac45e8c6f717bbd72a1702f7cd03920902259a1/eng/common/templates/job/job.yml#L68  But, it's probably better to add a `skipComponentGovernanceDetection` _parameter_ in default-build.yml and include `ne(parameters.skipComponentGovernanceDetection, 'true')`. If you do the second, would just need `skipComponentGovernanceDetection: true` at the same indentation as `artifacts:`.